# 0x62  data_package
Header header
uint8 tag
uint8 gw_id
uint8 node_num
Imu_data_package[16] node_data

